pkgs/by-name/sy/sync_music/package.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, ffmpeg
+, mutagen
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "sync_music";
+  version = "0.5.0";
+  format = "setuptools";
```

https://github.com/NixOS/nixpkgs/pull/254136

```suggestion
  pyproject = true;
```